apps/base/tasks.py

```diff
@@ -26,3 +30,39 @@ def run(self, *args, **kwargs):
         logger.info('Calling django management command to clear expired sessions')
 
         call_command('clearsessions')
+
+
+class BackupDatabaseTask(MoodyPeriodicTask):
+    run_every = crontab(minute=0, hour=3, day_of_week=0)
+
+    def delete_old_backups(self):
+        for backup_file in os.listdir(settings.DATABASE_BACKUPS_PATH):
+            if any([backup_file.startswith(model) for model in settings.DATABASE_BACKUP_TARGETS]):
```

Comment:
Change this to a filter of files in the backup directory that start with the name of one of the models we target and delete files from that. We should be iterating over a list of files in the backup directory that match backup filenames and deleting those files.